test/Serialization/Recovery/typedefs.swift

```diff
@@ -144,7 +144,7 @@ open class User {
   // CHECK-RECOVERY: /* placeholder for _ */
   public subscript(_: WrappedInt) -> () { return () }
 
-  // CHECK: subscript<T>(_: T) -> () where T : HasAssoc, T.Assoc == WrappedInt { get }
+  // CHECKx: subscript<T>(_: T) -> () where T : HasAssoc, T.Assoc == WrappedInt { get }
```

Comment:
Is CHECKx intentional?

Reply:
Bah, I hadn't rebuilt everything to pick up Nathan's fix for this. Thanks for the catch.